README.md

```diff
@@ -1,10 +1,10 @@
 <!-- based on template now, remaining parts marked as FILL TEMPLATE:  -->
 
-# Intel PMEM-CSI plugin for Kubernetes
+# Intel PMEM-CSI for Kubernetes
```

Comment:
After removing "plugin" in this sentence, i feel like lost its meaning :)

Reply:
This isn't a sentence :-)

Trying to explain what PMEM-CSI is in the title of the document is bound to fail. The title just needs to tell the reader which document he is looking at. What the thing is that the document describes is explained in the introduction.

Reply:
fair enough.